packages/cli/generators/relation/templates/controller-relation-template-has-one.ts.ejs

```diff
@@ -0,0 +1,37 @@
+import {
+  repository,
+} from '@loopback/repository';
+import {
+  param,
+  get,
+  getModelSchemaRef,
+} from '@loopback/rest';
+import {
+  <%= sourceModelClassName %>,
+  <%= targetModelClassName %>,
+} from '../models';
+import {<%= sourceRepositoryClassName %>} from '../repositories';
+
+export class <%= controllerClassName %> {
+  constructor(
+    @repository(<%= sourceRepositoryClassName %>) protected <%= paramSourceRepository %>: <%= sourceRepositoryClassName %>,
+  ) { }
+
+  @get('/<%= sourceModelPath %>/{id}/<%= targetModelName %>', {
+    responses: {
+      '200': {
+        description: '<%= targetModelClassName %> belonging to <%= sourceModelClassName %>',
+        content: {
+          'application/json': {
+            schema: {type: 'array', items: getModelSchemaRef(<%= targetModelClassName %>)},
```

Comment:
`schema: {type: 'array', items: getModelSchemaRef(<%= targetModelClassName %>)},`
should be
`schema: getModelSchemaRef(<%= targetModelClassName %>),`